packages/cli/src/commands/codegen.ts

```diff
@@ -30,8 +30,10 @@ export class CodeGen extends Command {
 
       if (hasPrisma && hasPrismaSchema) {
         let prismaSpinner = log.spinner(`Generating Prisma client`).start()
-        void runPrisma(["generate"], true).then((success) =>
-          success ? prismaSpinner.succeed() : prismaSpinner.fail(),
+        void runPrisma(["generate"], true).then((result) =>
+          result.success
+            ? prismaSpinner.succeed()
+            : prismaSpinner.fail() && console.log("\n" + result.stderr)
```

Comment:
Read first: [packages/cli/src/commands/prisma.ts:27](https://github.com/blitz-js/blitz/pull/2891/files#diff-28ae2cc5cebe012f80c6c602a99837c65b147f773ca87d1b029691e11b9b98c7R27)

When prisma fails, we take the captured stderr and print them using `console.log`.
Why `console.log` and not `log.*` alternatives?
Because `console.log` keeps the stderr generated by prisma and I think it's very well formated.

Here is a comparison of different output formats: [view comparison](https://user-images.githubusercontent.com/38964964/139505247-3ce439f3-2bbc-4584-828b-e423e395cc05.png)

The final result: 
![Screenshot_2021-10-29_17-18-46](https://user-images.githubusercontent.com/38964964/139505298-31c4bb3a-130f-4971-afe2-2d229e4cff91.png)

Reply:
Wow! Thanks so much for evaluating all the options!